app/components/Header/Header.module.scss

```diff
@@ -0,0 +1,14 @@
+@use '@/styles/variables.scss' as var;
+@use '@/styles/mixins.scss' as mix;
+
+.header {
+  @include mix.flex(center, center, column);
+  gap: var.$spacing-l;
+  padding: var.$spacing-l 0;
```

Comment:
Please use padding-block, since you need to set only top and bottom and default padding is 0